spec/definitions/DigitalWalletProvider.yaml

```
@@ -1,4 +1,4 @@
-description: Провайдер электронных денежных средств
+description: Провайдер электронных денежных средств (напр. qiwi)
```

(Здесь и далее) Быть может здесь добавить ссылку на то место, откуда их вообще можно узнать? Типа такого:
```suggestion
description: |
    Провайдер электронных денежных средств.
    
    Набор доступных для проведения платежей провайдеров можно узнать, вызвав соответствующую [операцию](#operation/getInvoicePaymentMethods) после создания инвойса.
example: qiwi
```

---

Fixed